hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1259,10 +1280,22 @@ public ListKeysResult listKeys(String volumeName, String bucketName,
           break;
         }
       }
+      stopNano = Time.monotonicNowNanos();
     }
-
     boolean isTruncated = cacheKeyMap.size() > maxKeys;
-
+    if (perfMetrics != null) {
+      long averagePagination;
+      if (isTruncated) {
+        averagePagination = maxKeys;
+      } else {
+        averagePagination = cacheKeyMap.size();
+      }
+      perfMetrics.setListKeysAveragePagination(averagePagination);
+      long opsPerSec =
+          averagePagination / (Time.monotonicNowNanos() - startNanos);
+      perfMetrics.setListKeysOpsPerSec(opsPerSec);
```

Comment:
Without conversion this looks more like "ops per nanoseconds".

Also, can reuse `stopNano` instead of getting another timestamp?

Reply:
changed its name.